src/modules/LanItems.rb

```diff
@@ -1229,7 +1229,7 @@ def BuildBondIndex
       deep_copy(index)
     end
 
-    def startmode_overview
+    def startmode_overview(item_id)
```

Comment:
\# @param item_id [Integer] a key for {#Items}

I keep forgetting which API uses numbers and which uses the kernel names.